crates/holochain_metrics/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+const DASH_NETWORK_STATS: &[u8] = include_bytes!("dashboards/networkstats.json");
+
+/// Initialize metrics. Call this from binaries, not libraries.
+pub async fn initialize_metrics(path: std::path::PathBuf) {
```

Comment:
does this rely on an influxdb instance running while the metrics are consumed and set up? i'm thinking of a way to store and asynchronously process them for batch testruns

Reply:
No, this runs influxdb as a child process of holochain. You can view the ui while holochain is running. Or if we're in a CI context we can ship the influx data directory as an artifact and run influx again later to inspect the data that was generated during the run.